### ISSUE TITLE
Sign Out Remains for Multiple Years

### ISSUE THREAD

**Issue by EarthSchlange** [usgo/gocongress#237]
## Summary of the Bug
It seems unusual that we are checking the year of our user for Sign In, but not if they are the correct year on other years.

## Steps to Reproduce the Behaviour
1. Sign In with any account.
2. Goto another year.

## The Expected Behaviour
The Sign Out button should only be for the current year. Previous years should require separate Sign In.

### 2018
![Screenshot from 2021-06-20 16-13-00](https://user-images.githubusercontent.com/5054653/122691490-d3bc1700-d1e4-11eb-9dd6-35bb5f941457.png)

### 2020
![Screenshot from 2021-06-20 16-13-09](https://user-images.githubusercontent.com/5054653/122691491-d454ad80-d1e4-11eb-8300-abe309e32ecc.png)

### 2021
![Screenshot from 2021-06-20 16-31-57](https://user-images.githubusercontent.com/5054653/122691530-1251d180-d1e5-11eb-96a4-79fb7c84be84.png)

**Comment by neagle** [comment]
Yeah, this is definitely a slightly funny aspect of the way the site works. I'm not sure it's a _problem_, though, especially since previous years are mainly kept around for historical reference.

To fix it, we could...

1. Update the site so that one could be logged in to any number of years simultaneously -- that login status would be year-specific and not site-specific, and the site would really act like different years were fundamentally different sites. This is a bigger technical challenge than I know how to handle at the moment. :) (@jaredbeck?)

2. Log a person out when they visit another year, which seems undesireable.

3. ?

The mix of difficulty + low urgency from my perspective has always made me leave this on the pile.
